### PR TITLE
 Introduce Microsoft / Azure mono repos

### PR DESCRIPTION
**Pull request by tomkerkhove**
Introduce Microsoft / Azure mono repos
- https://github.com/Azure/azure-sdk-for-net
- https://github.com/Azure/azure-storage-net
- https://github.com/Azure/azure-libraries-for-net
- https://github.com/aspnet/AspNetWebStack

### CODE REVIEW

File: packages/renovate-config-monorepo/package.json [hunk 1]
```diff
@@ -1028,5 +1056,8 @@
         "https://github.com/googlechrome/workbox"
       ]
     }
+  },
+  "dependencies": {
+    "gh-got": "^8.1.0"
```
Comment (rarkins) [inline packages/renovate-config-monorepo/package.json]: Any idea where this came from?

Comment (tomkerkhove) [reply packages/renovate-config-monorepo/package.json]: That's my bad actually given I had to install it to be able to run the command

Comment (tomkerkhove) [reply packages/renovate-config-monorepo/package.json]: Will remove it

**Comment by rarkins** [comment]
Thanks @tomkerkhove. This will be live in the app once I publish the groups to npm later today.

**Comment by tomkerkhove** [comment]
Thank you for the quick help, once again!

**Comment by rarkins** [comment]
It should now be live in the next 5-15 minutes depending on caches, and be applied for anyone who extends `config:base` or `group:monorepos`

**Comment by tomkerkhove** [comment]
Awesome, thanks!

**Pull request merged**